packages/reapit-connect/src/components/hocs/error-boundary.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
```

Comment:
I can't see where does it use. So if you don't use pls remove it, or Add it to Router instead of each page